src/Entities/User.php

```diff
@@ -34,7 +34,6 @@ class User extends Entity
         'created_at',
         'updated_at',
         'deleted_at',
-        'last_active',
```

Comment:
Please revert. Removing only `last_active` is not consistent.
And we want to advance OOP.

Reply:
Done!